app/javascript/app/pages/country-context/socioeconomic-indicators/shared/bar-chart-tooltip/bar-chart-tooltip-component.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from 'react';
+import Proptypes from 'prop-types';
+import styles from './bar-chart-tooltip-styles.scss';
+
+class CustomTooltip extends PureComponent {
+  render() {
+    const { content, config } = this.props;
+    const hasContent = content && content.payload && content.payload.length > 0;
+    const payload = content &&
+      content.payload &&
+      content.payload[0] &&
+      content.payload[0].payload;
```

Comment:
Tomek recently pointed out a cool function from lodash --> we could make it more readable by using `get` like so: `get(content, 'payload[0].payload')`